build_iso.sh

```diff
@@ -13,34 +21,79 @@ packages=(
 	python-pyparted
 )
 
-mkdir -p /tmp/archlive/airootfs/root/archinstall-git
-cp -r . /tmp/archlive/airootfs/root/archinstall-git
-
-cat <<- _EOF_ | tee /tmp/archlive/airootfs/root/.zprofile
-	cd archinstall-git
-	rm -rf dist
-
-	python -m build --wheel --no-isolation
-	pip install dist/archinstall*.whl
-
-	echo "This is an unofficial ISO for development and testing of archinstall. No support will be provided."
-	echo "This ISO was built from Git SHA $GITHUB_SHA"
-	echo "Type archinstall to launch the installer."
-_EOF_
-
 pacman -Sy
 pacman --noconfirm -S git archiso
```

Comment:
Perhaps you can add the `--disable-download-timeout` flag here. It helps with errors like `Operation too slow`.